js/everything.js

```diff
@@ -490,13 +508,17 @@ $(function() {
     // Handle form submission
     $("#form").submit(function(event) {
         event.preventDefault();
-
         var formValue = $.trim($("#v").val());
+        var formValueTime = formValue.match(/(&t=\d*)$/g);
```

Comment:
So basically we have two cases:

1. User inputs time position in the URL, `getParameterByName(formValue, "t");` retrieves that time position.

2. User inputs time position in the form, this section of the code deals with that.